### ISSUE TITLE
[CAIA A11y Research]: Core Identity Team, Terms of Use - Figma Prototype Testing with AT (mid-August 2023)

### ISSUE THREAD

**Issue by claytonzook** [department-of-veterans-affairs/va.gov-team#62359]
### Who's on your team?

```bash
Team name: Core Identity
OCTO-DE product owner: Tom Black
Product manager: Amber Parsley and Mike King
Designer: Alex Garcia and Clayton Zook as researcher
FE engineer: Alex Garcia
Accessibility specialist: none
Content writer: Tinu Awoliyi
Product/team Slack channel: #identity-surge 
(Notes indicate CAIA to use: #identity-ux channel) 

...
```

### Do you have an accessibility champion on your team?

No, we don't have a champion (yet)

### If you have an accessibility champion, what's their name?

Is this an official thing? If so then no one. Otherwise Clayton Zook

### Calendar checking

- [X] Based on this calendar, confirm we have specialists who can support your study in your desired timeframe
- [X] Confirm you are filing this support ticket at least 1 week in advance of said timeframe, so we have enough heads up to provide support

### Have you reviewed the CAIA capacity calendar?

- [X] Yes, we have reviewed the CAIA capacity calendar and can confirm specialists are available to support in our research timeframe

### When is the timeframe for your testing?

Mid August to late August (we are likely a week or two away from submitting to Research Review touchpoint)

### Supporting artifacts (if any)

- Product outline: [Product info](https://github.com/department-of-veterans-affairs/va.gov-team/tree/f180bf496a0b9b06b71770f20a4091a3b9bc7b04/products/identity/terms%20and%20conditions)
- User flows: [User flow in Figma](https://www.figma.com/proto/Fhw9P7r6pyaCGXRmolruoQ/Terms-of-Use?page-id=0%3A1&node-%5B=&node-id=58-976&starting-point-node-id=1%3A493&show-proto-sidebar=1)
- Prototype or mockups: [User flow in Figma](https://www.figma.com/proto/Fhw9P7r6pyaCGXRmolruoQ/Terms-of-Use?page-id=0%3A1&node-%5B=&node-id=58-976&starting-point-node-id=1%3A493&show-proto-sidebar=1)
- Research plan: [Research Folder in GH](https://github.com/department-of-veterans-affairs/va.gov-team/tree/91db02e7caf27417583e829894628cb7b3753e50/products/identity/Research/2023-07%20Terms%20of%20Use)
- [Research conversation guide](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/identity/Research/2023-07%20Terms%20of%20Use/2023-08%20Terms%20of%20Use%20conversation%20guide.md) 

...

### Next steps

- [x] Post a link to this ticket in the accessibility-help Slack channel and tag Terry Nichols.
- [x] Your team's researchers and observers read through platform guidance on research with assistive technology users
- [x] CAIA Specialists are assigned to your research, be sure to add the respective a11y specialist(s) to your session invites (include the Zoom or Teams link).
- [x] CAIA Specialists review your conversation guide and research plan
- [x] Provide specialists your participant details like skill level, operating system, assistive tech used, browser
- [x] CAIA specialists will provide you an optional feedback form to let us know how we can support y'all better into the future.
- [x] CAIA Specialists create a research notes template
- [x] [CAIA Post research, specialists will add accessibility findings to your CAIA folder](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/teams/CAIA/accessibility/Deliverables/Core%20Identity%20Team/2023-Aug-accessibility-findings-Identity-Core.md)
    - While this study had minimal AT participants three (3), with two (2) of which who used their technology, CAIA was able to capture valuable feedback, which was shared in the Mural for this project. 
    - Please find this deliverable template we are testing out, which dives deeper into the findings we shared in the Mural.

**Comment by sara-amanda** [comment]
## CAIA Update
**_Looking forward to your research @claytonzook and team!_** 

Happy to help with the [assistive technology sessions](https://depo-platform-documentation.scrollhelp.site/research-design/research-assistive-technology-sessions) with @AngelaFowler82. I see your note re: timing and will watch for an update, when that information becomes available. 

> Mid August to late August (we are likely a week or two away from submitting to Research Review touchpoint)

cc: @coforma-terry

**Comment by coforma-terry** [comment]
Hey @claytonzook  - QQ for you - I'm not able to see the #identity-surge channel in DSVA - is it under a different name now?
Thanks!

**Comment by claytonzook** [comment]
@coforma-terry Our channels are private since there's some sensitive sign in stuff discussed. Let's do any chatting about our research in #identity-ux channel. I'll send you an invite!

**Comment by coforma-terry** [comment]
Perfect - thank you! Can you also invite @sara-amanda too?

**Comment by sara-amanda** [comment]
@claytonzook when you add @coforma-terry and myself, can you also add @AngelaFowler82? Thanks so much in advance!

**Comment by sara-amanda** [comment]
## Accessibility Champs

> **If you have an accessibility champion, what's their name?**
> Is this an official thing? If so then no one. Otherwise Clayton Zook

[Yes, @claytonzook! Here is a link to the Accessibility Champs Program.](https://github.com/department-of-veterans-affairs/va.gov-team/tree/master/teams/vsa/accessibility/a11y-champs#accessibility-champions-community) Your comment brings up an excellent point. I will go ahead and link to the program from the research ticket, for those who are not familiar with the newly, launched program. 

### About A11y Champs
- It is easy to participate. Anyone working at the VA, with access to the DSVA slack channel, can become an accessibility champion through this easy, online curriculum. ♥️
- This is a grassroots, volunteer-led educational program intent on scaling up VA.gov's focus on the disabled Veteran experience through: co-design, curiosity, and community;  centering the diverse voices of disabled Veterans, caretakers, and family members; and asking questions first and learning from others.
- Join the Slack Channel:[ #a11y-champs](https://dsva.slack.com/archives/C0552U2L30S)

cc: @joshkimux

**Comment by coforma-jamie** [comment]
Can I get an invite as well @claytonzook? Thank you!

**Comment by sara-amanda** [comment]
## CAIA Feedback on Figma Prototype Inquiry

Following up on our [Slack Thread from 7/25 and 7/26/2023](https://dsva.slack.com/archives/C8E985R32/p1690294453381359?thread_ts=1689881721.490029&cid=C8E985R32) here, since @claytonzook is OOO. 

### Figma Prototypes :thumbsup: 
So, I have heard that too from others that they traditionally wait until they are in a staging environment to test (a lot use Sketch for prototyping); however, _you said the magic word_ "**Figma**," which **should be compatible with VO, Jaws and NVDA screen readers**. 

I say "should be"  because, while possible to test with AT, it **requires some prep work on your end, first, to make sure it will work properly.** More on that below ... 

### Accessible Prototypes

> While in `accessibility mode`, Figma design elements are translated to HTML or HyperText Markup Language.  Screen readers interpret HTML tags to understand the content and sections of a document, as well as which elements are available for a user to interact with on a page. Semantic HTML tags also allow a screen reader to communicate content that may be highlighted visually to a user.
> 

[This article describes how to activate accessibility mode. ](https://help.figma.com/hc/en-us/articles/7810391964695-Accessible-prototypes-in-Figma#Activate_accessibility_mode)

### Mobile Not Included (For Now) :no_mobile_phones: 

> **Accessible prototypes are only available in the desktop browser and on the Figma desktop app.** 
> **Accessibility mode does not work for mobile or mobile web prototypes.**
#### Questions
- Are you planning to test mobile and desktop? 
 - If so, you would only be able to use the prototypes for desktop. 

#### Browser and Operating System Requirements (Desktop Only) 🖥️ 

**The minimum browser requirements are:**
- Chrome 72 or later
- Firefox 78 or later 
- Safari 14.1 or later
- Microsoft Edge 79 or later

**The minimum operating system (OS) requirements are:**
- Windows 8.1 or later
- Apple macOS 11 or later
- Any Linux OS that supports the browsers mentioned above
- Any Chrome OS that supports the browsers mentioned above

[Learn more about browser and OS requirements. ](https://help.figma.com/hc/en-us/articles/360039827194-Figma-browser-requirements)

### Words of caution from [someone who tried it (ref. article)](https://bootcamp.uxdesign.cc/i-tried-the-prototype-screen-reader-in-figma-beta-ceda511d715a) :warning: 

#### Build your prototype with HTML in mind

> If you want to test with the screen reader, you need to be very careful in the way you name your components and frames, keeping HTML tags in mind and assigning meaningful labels.
> Besides tags, another thing you need to pay attention to is focus order. Elements inside a frame may look fine even if the order is not correct, but screen readers will scan the page left to right, top to bottom, so all elements must have a clear hierarchical order.

#### Evaluate your user flow carefully

> Considering only elements with an interaction attached are read by the screen reader, are there elements in the interface that are not interactive? What kind of experience would a user have if they are not aware of these elements?
> You need to evaluate whether the type of user flow you are looking to test contains elements without an On Tap/Click interaction and whether this means a different experience for someone using the prototype screen reader compared to someone who is not using it.

#### Try it yourself

> Before starting to recruit participants for your user test, make sure everything works as expected. 
> Take the time to turn on the screen reader and test it yourself first.

cc: @coforma-terry and @coforma-jamie

**Comment by sara-amanda** [comment]
## CAIA Calendar
Research dates to be obtained (TBD) on, around or by 8/15/2023 according to the Epic.
- [ ] When dates are known, we will need to block time on the CAIA Research Calendar.
cc: @coforma-terry

**Comment by claytonzook** [comment]
@sara-amanda I have our [Research Review request](https://github.com/department-of-veterans-affairs/va.gov-research-repository/issues/351) in!

*   If we can do a practice screen reader sesh Wed-Fri Aug 9th-11th, that'd be amazing!
*   Assuming research review is completed on Mon, sessions could begin as early as Aug 15-16, we have offered calendar availability to the recruiting team through Aug 25th. We've requested 5 completed sessions
*   I'll plan to let you know, [in slack channel identity-research](https://dsva.slack.com/archives/C04S98PE8FK) if that's ok, as we get sessions scheduled with screen reader participants, so your team can plan who is joining those
*   I'm drafting a [separate convo guide](https://github.com/department-of-veterans-affairs/va.gov-team/blob/e69725bb1be3e1e91fc593e9e0d464e20f32ad54/products/identity/Research/2023-07%20Terms%20of%20Use/2023-08%20Terms%20of%20Use%20convo%20guide%20-%20screen%20reader.md) for these screen reader sessions that will address some circumastances that will be different with these sessions, I'll have this finished Aug 7/8
*   [Our Figma prototype](https://www.figma.com/proto/Fhw9P7r6pyaCGXRmolruoQ/Terms-of-Use?type=design&node-id=381-31&t=6Iu86HTNhe9kB8yC-0&scaling=scale-down&page-id=0%3A1&starting-point-node-id=381%3A31&show-proto-sidebar=1) is largely ready for screen reader use - just a few more tweaks to make there Aug 7/8
*   Anything else I should be considering at this point?

cc: [@coforma-terry](https://github.com/coforma-terry)

**Comment by coforma-jamie** [comment]
AT testing schedule:

* **Wednesday, August 16⋅3:00 – 4:30pm EST** 
    Identity Research, AT Desktop (P4 <> Clayton) 
    * Sara attending
    * :arrow_right: CAIA Team Member: Jamie
    * SK attending 
* **Thursday, August 17⋅3:00 – 4:30pm EST** 
    Identity Research, AT Desktop (P8 <> Clayton) 
    * Sara* 
        (I have a hard stop at 4 p.m. for an Oddball All Hands Meeting, if anyone else can join and stay the whole time? I can take notes until I have to jump.)
    * :star: :arrow_right: CAIA Team Member: Jamie :star: 
* **Friday, August 18⋅3:00 – 4:30pm EST**
    Identity Research, AT Desktop (P12 <> Amanda) 
    * Sara PTO OOO - but could try and join.
    * :arrow_right: CAIA Team Member: Jamie
* **Monday, August 21⋅3:00 – 4:30pm**
    Identity Research, AT Desktop (P15 <> Charlye) 
    * Sara attending
    * :arrow_right: CAIA Team Member: SK 
    * Kate

**Comment by sara-amanda** [comment]
## CAIA Update: Revised AT testing schedule

### Wednesday, August 16⋅3:00 – 4:30pm EST

- [x] Status `No Show`

 * **Type**: Identity Research
    * **Environment:AT Desktop** 
    * **AT Type** (i.e. screen reader): 
    * **Participant**: P4
    * **Researcher:** Clayton
    * **CAIA Attendees:** 
        * Sara
        * Jamie
        * SK
   
### Thursday, August 17⋅3:00 – 4:30pm EST
- [ ] Status `Scheduled`
 * **Type**: Identity Research
    * **Environment:AT Desktop** 
    * **AT Type** (i.e. screen reader): 
    * **Participant**: P8
    * **Researcher:** Clayton
    * **CAIA Attendees:** 
        * Sara
        * Jamie
### Friday, August 18⋅3:00 – 4:30pm EST
- [ ] Status `Scheduled`
 * **Type**: Identity Research
    * **Environment:AT Desktop** 
    * **AT Type** (i.e. screen reader): 
    * **Participant**: P12
    * **Researcher:** Amanda
    * CAIA Attendees: 
        * Sara - PTO
        * Jamie
### Monday, August 21⋅3:00 – 4:30pm
- [ ] Status `Scheduled`
 * **Type**: Identity Research
    * **Environment:AT Desktop** 
    * **AT Type** (i.e. screen reader): 
    * **Participant**: P15
    * **Researcher:** Charlye
    * **CAIA Attendees:** 
        * Sara
        * Kate
        * SK attending

**Comment by claytonzook** [comment]
Unfortunately, we need to end session. I've asked the recruiters not to schedule anyone else and to close out this study at this point. I appreciate all the help! We'll look forward to more successful sessions next time.

**Comment by claytonzook** [comment]
Unfortunately, we need to end session. I've asked the recruiters not to schedule anyone else and to close out this study at this point. I appreciate all the help! We'll look forward to more successful sessions next time.

**Comment by sara-amanda** [comment]
## CAIA Update: Research Synthesis

- CAIA a11ys will be holding a sync on 9/12/2023 to go over findings. 
- We plan to share our official AT research feedback by the end of the week. 

cc: @ohgreatkate @coforma-jamie @SarahKay8
